client/core.tsx

```
@@ -1,3 +1,5 @@
+import { NextResponse } from "next/server";
```

ESLint

[error] 'NextResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)